js/apps/account-ui/public/locales/en/translation.json

```diff
@@ -97,7 +98,7 @@
   "removeConsentError": "Could not remove consent due to: {{error}}",
   "removeConsentSuccess": "Successfully removed consent",
   "removeCred": "Remove {{0}}",
-  "removeModalMessage": "Hiermee wordt de momenteel verleende toegangsrechten voor {{0}} verwijderd. U moet opnieuw toegang verlenen als u deze app wilt gebruiken.",
+  "removeModalMessage": "This will remove the currently granted access permission for {{0}}. You will need to grant access again if you want to use this app.",
```

Comment:
Looks like I accidentally changed this label to Dutch in a previous PR, so I've fixed it here.